samples-generator/specs/bin/minimalTriangleGLTF.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+module.exports = {
```

Comment:
I'd kinda rather this just be a file on disk, especially if we start adding more glTF test cases.

Reply:
It could go in a new folder `specs/data`